src/main/java/uk/gov/hmcts/reform/et/syaapi/notification/NotificationsProperties.java

```diff
@@ -30,6 +30,10 @@ public class NotificationsProperties {
     @NotEmpty
     private String submitCaseEmailTemplateId;
 
+    @Value("${notifications.cySubmitCaseEmailTemplateId}")
+    @NotEmpty
```

Comment:
Not sure about Business logic, but maybe it is better to use @NotBlank which will check null, emptiness and empty spaces?

Reply:
good point, changed it for all properties. Thanks